### PR TITLE
Few fixes for stable branching stage

### PR DESCRIPTION
**Pull request by mkolesnik**
1. Fix images script not to run at this stage

2. Fix shipyard handling to cover all projects, since all use shipyard

3. Use subctl devel in the initial shipyard image, since it's not
   expected to be available

Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

### CODE REVIEW

**Review by dfarrell07 — APPROVED** [review]

**Comment by nyechiel** [comment]
@mkolesnik this is conflicting now

File: releases/target [hunk 1]
```diff
@@ -1 +0,0 @@
-v0.8.1.yaml
```
Comment (mangelajo) [inline releases/target]: I really think we should stop using the link, and auto-detect the yaml based in "HEAD" changes.

**Review by mangelajo — APPROVED** [review]

**Comment by mangelajo** [comment]
oh conflict :/

**Review by tpantelis — APPROVED** [review]

File: releases/target (hunk 1 above)
Comment (mkolesnik) [reply releases/target]: We could but the only problem I see with that is if someone updates an older file (or an example file) and the script will think it needs to be released.

Also it'll make local testing a bit harder.

**Pull request merged**